### DISCUSSION TITLE
Student developer pack

### DISCUSSION THREAD

**Discussion by Usama084** [community/community#36316] [GitHub Education] [answered]
I was unable to do the verification of my student documents

**Comment by DSDmark** [comment]
Hey @Usama084,
I think you are talking about your **developer student pack**
About for verification, I think you should apply it again with your email normal, which you use normally with any official document of your school or collage or try to use another document. Just try and try it.

> Maximum it takes 1 week for verifying your document or email.

### There are several reasons why your verification is Rejected

#### The item you uploaded is insufficient to demonstrate your current academic status. Modifications to your photo proof might have been detected by the system. Please see suggestions.

- Suggested proof to expedite your approval:

- School issued photo ID with current enrollment date.

 - Schedule for current enrollment with school name.

 - Signed registration certificate with current enrollment date.

-  Transcripts with current enrollment date.

**Comment by ettaboyle** [comment, marked as answer]
Hi @Usama084 Thanks for reaching out! You can find all the details you need about the GitHub Student Developer Pack in this post:
[Student Developer Pack Application & FAQs ➡️ Start Here](https://github.com/orgs/community/discussions/17814)

Please review all the information in that post and follow the steps outlined. Thank you!

**Comment by sanah-naik** [comment]
I don't have an email address issued by my school. Is there any other way out

**Reply by Wishy-S** [reply]
I think mail is must

**Comment by basurajp** [comment]
You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote.Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.



i am facing these error mine id card mention  its online but portal Is not accepting
